storybooknative/src/types/Style.js

```diff
@@ -10,11 +10,23 @@
 import { type ____DangerouslyImpreciseStyle_Internal } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
 import AnimatedNode from "react-native/Libraries/Animated/src/nodes/AnimatedNode";
 
-type ExactStyles = $Exact<____DangerouslyImpreciseStyle_Internal>;
+// The fontSize style property should be a string for the react-native-web
+type ExactStyles = $Exact<{
+  ...____DangerouslyImpreciseStyle_Internal,
+  +fontSize?: string,
+  +borderWidth?: number | AnimatedNode | string,
+  +lineHeight?: number | string,
+}>;
 type ShapedStyles = $Shape<ExactStyles>;
 
 export type Style = {|
   ...ShapedStyles,
+  +ios?: ExactStyles,
+  +android?: ExactStyles,
+  +web?: ExactStyles,
+|};
+
+export type OutputStyle = {|
+  ...ExactStyles,
   +borderWidth?: number | AnimatedNode | string,
-  +lineHeight?: number | string,
```

Comment:
I am wondering whether we should have the same:
```
  +fontSize?: string,
  +borderWidth?: number | AnimatedNode | string,
  +lineHeight?: number | string,
```

in `OutputStyle` too?

Reply:
They are inside because it is part of `ExactStyles` now. 
But there is this confusing and not necessary type `ShapedStyles ` so I will remove it.

Reply:
So maybe we can even remove `+borderWidth?: number | AnimatedNode | string,` too?

Reply:
Good point. Thank you